featuregate/client/pkg/featuregateclient/client.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package client
+package featuregateclient
 
 import (
 	"context"
```

Comment:
Question: Do we have a plan to refactor `"github.com/vmware-tanzu/tanzu-framework/pkg/v1/config"` and `"github.com/vmware-tanzu/tanzu-framework/apis/config/v1alpha1"` ? I see those two imports in few lines below. 

(I was confused on the diffs between those two `config` until I looked into the code)